rllib/agents/slateq/slateq.py

```diff
@@ -113,21 +106,17 @@
     "training_intensity": None,
 
     # === Optimization ===
-    # TODO: Unify torch and tf config settings:
-    # Learning rate for adam optimizer for the user choice model.
-    "lr_choice_model": 1e-3,  # Only relevant for framework=torch.
-    # Learning rate for adam optimizer for the q model.
-    "lr_q_model": 1e-3,  # Only relevant for framework=torch.
-    # TODO: Unify torch and tf config settings:
-    # Learning rate for adam optimizer for the q model.
-    "lr": 0.00025,  # Only relevant for framework=tf|tf2.
+    # Learning rate for RMSprop optimizer for the q-model.
+    "lr": 0.00025,
```

Comment:
this is pretty low -- is this to avoid exploding q values? Are we using target q networks for regularization in torch/tf anymore?